common/djangoapps/monkey_patch/third_party_auth.py

```diff
@@ -26,3 +29,14 @@ def _create_social_auth(*args, **kwargs):
         return classmethod(_create_social_auth)
 
     DjangoUserMixin.create_social_auth = create_social_auth_wrapper(DjangoUserMixin.create_social_auth)
+
+
+def patch_cms():
```

Comment:
@symbolist When I started up Studio via 'paver devstack studio', I saw the python_social_auth warnings for these models. But when I started up LMS via 'paver devstack lms', I did not see the warnings. So that's why I only monkey patched CMS. I agree that both LMS & CMS uses the `social` module - any ideas why I'd see the warnings in one versus the other?